galaxy_ng/app/api/ui/serializers/execution_environment.py

```diff
@@ -284,6 +286,11 @@ class Meta:
     def get_write_only_fields(self, obj):
         return utils.get_write_only_fields(self, obj)
 
+    def get_is_indexable(self, obj):
+        if obj.get_registry_backend():
+            return True
+        return False
```

Comment:
```suggestion
        return bool(obj.get_registry_backend())
```

has the same effect

Reply:
I kinda like the more explicit `if... then` it's easier to read.